docs/guides/networking/linode-network/ip-failover/index.md

```diff
@@ -73,23 +73,30 @@ To configure failover, complete each section in the order shown:
+{{</ note >}}
+
 1.  Disable Network Helper on both instances. For instructions, see the [Network Helper](/docs/guides/network-helper/#single-per-linode) guide.
 
-1. Add an additional IPv4 address to one of the Compute Instances. See the [Managing IP Addresses](/docs/guides/managing-ip-addresses/#adding-an-ip-address) guide for instructions. Make a note of the newly assigned IP address. *Each additional IP address costs $1 per month*.
+1. Add an additional IPv4 address _or_ IPv6 range (/64 or /56) to one of the Compute Instances. See the [Managing IP Addresses](/docs/guides/managing-ip-addresses/#adding-an-ip-address) guide for instructions. Make a note of the newly assigned IP address. *Each additional IPv4 address costs $1 per month*.
 
-1. On the *other* Compute Instance, add the newly assigned IP address as a *Shared IP* using Linode's *IP Sharing* feature. See [Managing IP Addresses](/docs/guides/managing-ip-addresses/#configuring-ip-sharing) for instructions on configuring IP sharing.
+1. On the *other* Compute Instance, add the newly assigned IPv4 address or IPv6 range as a *Shared IP* using Linode's **IP Sharing** feature. See [Managing IP Addresses](/docs/guides/managing-ip-addresses/#configuring-ip-sharing) for instructions on configuring IP sharing.
 
 ### Add the Shared IP to the Networking Configuration
 
 Adjust the network configuration file on *each* Compute Instance, adding the shared IP address and restarting the service.
 
 1.  Log in to the Compute Instance using [SSH](/docs/guides/connect-to-server-over-ssh/) or [Lish](/docs/guides/using-the-lish-console/).
 
-1.  Add the shared IP address to the system's networking configuration file. Within the instructions for your distribution below, open the designated file with a text editor (such as [nano](/docs/guides/use-nano-to-edit-files-in-linux/) or vim) and add the provided lines to the end of that file. Replace *[shared-ip]* with your own shared IP address.
+1.  Add the shared IP address to the system's networking configuration file. Within the instructions for your distribution below, open the designated file with a text editor (such as [nano](/docs/guides/use-nano-to-edit-files-in-linux/) or vim) and add the provided lines to the end of that file. When doing so, make the following replacements:
+
+    - **[shared-ip]**: The IPv4 address you shared or an address from the IPv6 range that you shared. You can choose any address from the IPv6 range. For example, within the range *2001:db8:e001:1b8c::/64*, the address `2001:db8:e001:1b8c::1` can be used.
```

Comment:
@wildmanonline do you think we should also use backticks instead of italics for the 2001:db8:e001:1b8c::/64 range

Reply:
My initial thought is to leave it as is. I used backticks on the specific address (and not the range) so it might be easier for users to understand what address they can actually use when running the commands / editing the files below. That address is a potential replacement value for that variable, whereas the range is not.

Reply:
👍